src/components/home/AboutSection.jsx

```diff
@@ -6,25 +6,27 @@ const AboutSection = () => {
   return (
     <div className="w-9/12 flex flex-col items-center text-center gap-10 mb-0">
       <Title text="About Us" />
-      <p>
-        Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod
-        tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim
-        veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea
-        commodo consequat. Duis aute irure dolor in reprehenderit in voluptate
-        velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint
-        occaecat cupidatat non proident, sunt in culpa qui officia deserunt
-        mollit anim id est laborum.
+      <p className="text-lg">
+        ArtFactory at UCR is a vibrant community of artists established in
+        spring 2023. Our mission? <br></br>
+        <b>
+          To cultivate creativity and exploration across various art forms.
+        </b>{" "}
```

Comment:
remove this description also and  pull from dev since this file should be called what.jsx